module/VuFindSearch/src/VuFindSearch/Backend/EDS/Backend.php

```diff
@@ -422,7 +435,12 @@ protected function paramBagToEBSCOSearchModel(ParamBag $params)
             $options[$key] = in_array($key, $arraySettings)
                 ? $param : $param[0];
         }
-        return new SearchRequestModel($options);
+        $model = new SearchRequestModel($options);
+        $model->setValidationConfig($this->validationConfig);
```

Comment:
Would it make sense to add an optional constructor parameter to the model in addition to or instead of this setter? I don't have strong feelings one way or the other; just wondering aloud.

Reply:
Yeah I considered that, there seem to be patterns in our codebase in both directions, but in retrospect I agree it should be there (in addition to the setter).  This is done.